mathesar/exception_handlers.py

```diff
@@ -0,0 +1,50 @@
+from rest_framework.views import exception_handler
+
+from mathesar.api.exceptions.error_codes import ErrorCodes
+
+exception_map = {
+}
```

Comment:
Is something missing here, or am I not understanding?

Reply:
I split error handling into multiple PR as I felt the original PR was huge, that is the reason for so many incomplete placeholders. This is supposed to be a last resort conversion layer which would map any non spec Exception into its equivalent ApiException, l will open a PR that would make use of this.